LICENSE.md

```diff
@@ -0,0 +1,6 @@
+Copyright 2019 - present
```

Comment:
`- present` is not valid in a copyright statement. Also, a copyright statement should include the legal name of the entity which is claiming that copyright.